lzy/allocator/src/main/java/ai/lzy/allocator/configs/ServiceConfig.java

```diff
@@ -18,12 +19,17 @@
 @ConfigurationProperties("allocator")
 public class ServiceConfig {
     private String instanceId;
-    private String address;
+    private List<String> hosts = new ArrayList<>(); // either [<ipv4>] or [<ipv4>,<ipv6>]
+    private Integer port;
     private Duration allocationTimeout;
     private Duration heartbeatTimeout;
     private List<String> serviceClusters = new ArrayList<>();
     private List<String> userClusters = new ArrayList<>();
 
+    public String getAddress() {
```

Comment:
Why last one?

Reply:
ipv6 has a higher priority than ipv4. IPs has [<ipv4>,<ipv6>] format, so we need the last one.

Reply:
let's do it more explicetly, plz
i.e. let's split it by `,` and parse each part into IPv4 or IPv4 object
and then select on of them (with reason description)